### PR TITLE
Add background context and reflection questions to Lori Danson simulation

### PR DESCRIPTION
**Pull request by kesiena115**
Here are screenshots of the changes:

screenshot 2017-05-31 14 47 52

screenshot 2017-05-31 14 48 03

screenshot 2017-05-31 14 48 14

screenshot 2017-06-01 01 26 17

screenshot 2017-05-31 15 26 36

### CODE REVIEW

**Comment by kesiena115** [comment]
I have not yet added the summary of responses at the bottom of the post-simulation reflections. The main reason is because when renderQuestionEl() gets called in LinearSession, it does not include previous questions & responses. I have some ideas on how to implement the desired screen, but it might be better for us to first discuss it.

**Comment by kevinrobinson** [comment]
@kesiena115 awesome!  just noting here what we talked about in person: I'd suggest seeing if you can modify `getNextQuestion` to also include the past responses for the reflection questions that require past responses as well as see how that goes.

**Comment by kevinrobinson** [comment]
@kesiena115 awesome!  just noting here what we talked about in person: I'd suggest seeing if you can modify `getNextQuestion` to also include the past responses for the reflection questions that require past responses as well as see how that goes.

**Comment by kesiena115** [comment]
@kevinrobinson it's ready for another review.

**Comment by kesiena115** [comment]
@kevinrobinson ready for another review

File: ui/src/message_popup/playtest/danson_experience_page.jsx [hunk 1]
```diff
@@ -122,64 +102,107 @@ export default React.createClass({
...
+            responsePrompt="Your Answer"
+            recordText="NEXT"
             onLogMessage={onLog}
+            onResponseSubmitted={onResponseSubmitted}
+          />
+        </div>
+      );
+    }
+
+    if (question.stage === 'postreflect') {
```
Comment (kesiena115) [inline ui/src/message_popup/playtest/danson_experience_page.jsx]: No. The difference is that we show the response summary under each postreflect question.

Comment (kevinrobinson) [inline ui/src/message_popup/playtest/danson_experience_page.jsx]: 👍

File: ui/src/message_popup/playtest/danson_experience_page.jsx [hunk 2]
```diff
@@ -122,64 +102,107 @@ export default React.createClass({
...
+          key={question.id}
+          question={question}
+          choices={question.choices}
+          onLogMessage={onLog}
+          onResponseSubmitted={onResponseSubmitted}
+          forceResponse={true}
+        />;
+      } else {
+        return (
+          <div>
```
Comment (kesiena115) [inline ui/src/message_popup/playtest/danson_experience_page.jsx]: It's not extra. I've fixed the indentation. The div is a container for PlainTextQuestion and MinimalOpenResponse

**Pull request merged**